scripts/generate_can.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python3
```

Comment:
This is the script that gets called for generating all the auto-generated CAN files. So after making a change to the YAML CAN file, you'd call `python3 scripts/generate_can.py` from the hw bridge root (using python3 cuz cantools isn't supported by python2) and it'll re-generate all the files under can/generated (and also update some parts of the YAML). 

This script reads from the YAML and uses the cantools API to create the DBC file, then invokes a cmd line cantools call to generate the C source and header from the DBC. The script then calls the generate_can_enums.py and generate_can_wrapper.py scripts to generate the uwrt_mars_rover_can_enums.h, uwrt_mars_rover_can_wrapper.h, and uwrt_mars_rover_can_wrapper.cpp files.